debian/control

```diff
@@ -107,7 +107,7 @@ Section: libdevel
 Architecture: linux-any
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
-Depends: libmircommon10 (= ${binary:Version}),
+Depends: libmircommon11 (= ${binary:Version}),
```

Comment:
Hmm, `debian/changelog` for 2.16 says "mircommon ABI unchanged at 9". And...

```
$ git diff v2.16.4 -- debian/libmircommon10.install
diff --git a/debian/libmircommon10.install b/debian/libmircommon10.install
new file mode 100644
index 0000000000..5233fdfb45
--- /dev/null
+++ b/debian/libmircommon10.install
@@ -0,0 +1 @@
+usr/lib/*/libmircommon.so.10
```

So 10 is already new in 2.17